qemu/tests/blk_stream.py

```diff
@@ -29,18 +33,36 @@ def start(self):
         start block device streaming job;
         """
         params = self.parser_test_args()
-        base_image = params.get("base_image")
         default_speed = params.get("default_speed")
 
         error.context("start to stream block device", logging.info)
-        self.vm.block_stream(self.device, default_speed, base_image)
+        self.vm.block_stream(self.device, default_speed, self.base_image,
+                             backing_file=self.backingfile)
         status = self.get_status()
         if not status:
             raise error.TestFail("no active job found")
         msg = "block stream job running, "
         msg += "with limited speed %s B/s" % default_speed
         logging.info(msg)
 
+    def get_base_backingfile(self):
+        """
+        Get values for base and backing-file option.
+        """
+        params = self.parser_test_args()
+        backingfile_option = params.get("backingfile_option", "no")
+        if backingfile_option == "no":
+            return
+
+        self.base_image = self.get_image_file()
```

Comment:
@luckyh Thanks. self.base_image is current image_file in this test case, but it can be other snapshot file in other test cases, I can modify it to self.base_image = self.image_file to make it more sensible here.